code/controllers/subsystem/stock_market.dm

```diff
@@ -13,14 +13,16 @@ SUBSYSTEM_DEF(stock_market)
 	var/list/materials_trend_life = list()
 	/// Associated list of materials alongside their available quantity. This is used to determine how much of a material is available to buy, and how much buying and selling affects the price.
 	var/list/materials_quantity = list()
+	/// A list of all currently active stock market events.
+	var/list/active_events = list()
 	/// HTML string that is used to display the market events to the player.
 	var/news_string = ""
 
 /datum/controller/subsystem/stock_market/Initialize()
 	for(var/datum/material/possible_market as anything in subtypesof(/datum/material)) // I need to make this work like this, but lets hardcode it for now
-		if(initial(possible_market.tradable))
+		if(possible_market::tradable)
```

Comment:
I'm not 100% sure this is the *right* place to use `::`, as per the style guide?
![image](https://github.com/tgstation/tgstation/assets/51863163/17a6848d-95f9-4577-95a6-80a9a3593347)

Reply:
Oof, good to know, also that means I've probably butchered its implementation in arcmining so that'll be something I'll need to fix